packages/core/src/scss/components/checkout/achForm/_achForm.scss

```diff
@@ -2,10 +2,15 @@
   display: flex;
   flex-flow: row wrap;
   margin: 0 -0.5 * $floating-form-field-spacing;
-  margin-bottom: spacing("half");
+  padding: (spacing("base") / 1.5) 0;
```

Comment:
Can we find a close value of `spacing()` to replace the math( / 1.5 ) here?

Reply:
Hey @bc-nick thanks for pointing this, does it make sense to make it a variable?

Reply:
Hey @bc-nick thanks for pointing this, does it make sense to make it a variable?

Reply:
@animesh1987
I will include this moment when I prepare the request about which I wrote here
https://github.com/bigcommerce/checkout-js/pull/1289#discussion_r1212799549